COMETwebapp/COMETwebapp/COMETwebapp.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
please add the following:

<Version>your version</Version>
<Title>your title</Title>
<Description>your description</Description>
<Company>RHEA System S.A.</Company>
<Copyright>Copyright 2022 RHEA System S.A.</Copyright>
<PackageLicenseExpression>AGPL-3.0-only</PackageLicenseExpression>
<Authors>Justine, Sam, Alex, Alexander</Authors>
<RepositoryType>Git</RepositoryType>
<RepositoryUrl>https://github.com/RHEAGROUP/COMET-WEB-Community-Edition.git</RepositoryUrl>